source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,176 @@
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Leave app running in notification center when application window is closed    | N/A                       | v1.3.0 +                  | v1.3.0 +                  | `magicmonty <https://github.com/magicmonty>`_ (OS X); `yuya-oc <https://github.com/yuya-oc>`_ (Linux)                                                  |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Show red badge on taskbar icon to indicate unread messages                    | v1.4.0 +                  | v1.4.0 +                  | N/A                       | `Razzeee <https://github.com/Razzeee>`_                                                                                                                |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Flash taskbar icon when a new message is received                             | v1.3.0 +                  | N/A                       | v1.4.0 +                  | `MetalCar <https://github.com/metalcar>`_ (Windows); `jnugh <https://github.com/jnugh>`_ (Linux)                                                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+**Hide menu bar**
+
+Hides the menu bar. When enabled, pressing ``ALT`` will toggle whether the menu bar is shown or hidden.
+
+This setting is off by default.
+
+**Allow mixed content**
+
+If your server is hosted on `https://`, insecure content, images and scripts with `http://` are not rendered by default. This option allows such content to be rendered. If you do enable it, please be mindful of potential security risks shared over `http://` protocols.
+
+This setting is on by default.
+
+Note: Enabling both secure and insecure content will break YouTube videos. An open issue for ``electron`` on which the app is built is under investigation: https://github.com/electron/electron/issues/2749
```

Comment:
This note on Electron seems Expert Mode, only developers will really understand this. Propose something like 
```
Note: Enabling mixed content will disable YouTube video previews [due to an issue in the underlying technology](link_to_electron_issue) used by the Mattermost Desktop app.
```